packages/grafana-ui/src/components/Logs/LogRows.tsx

```diff
@@ -138,7 +126,7 @@ class UnThemedLogRows extends PureComponent<Props, State> {
               onClickLabel={onClickLabel}
             />
           ))}
-        {hasData && deferLogs && <span>Rendering {rowCount} rows...</span>}
+        {hasData && !renderAll && <span>Rendering {rowCount - PREVIEW_LIMIT} rows...</span>}
```

Comment:
Found this, should be the same as above right?
```suggestion
        {hasData && !renderAll && <span>Rendering {rowCount - (previewLimit || PREVIEW_LIMIT)} rows...</span>}
```